packages/stratocacher-layer-dynamo/src/index.js

```diff
@@ -0,0 +1,100 @@
+import Q from "q";
+import {DynamoDB} from "aws-sdk";
+import {Layer} from "stratocacher";
+
+var client, awsConfig, tableName;
+
+class Client {
+	constructor(awsConfig) {
+		const ddbClient = new DynamoDB(awsConfig);
+		this.getItem = Q.nbind(ddbClient.getItem, ddbClient);
+		this.putItem = Q.nbind(ddbClient.putItem, ddbClient);
+	}
+}
+
+function getClient() {
+	if (!client) {
+		client = new Client(awsConfig);
+	}
+	return client;
+}
+
+/*
+	DynamoDB Caching Layer.
+	Uses AWS-SDK Dynamo client: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB.html
+	Configuration requires a table name corresponding to Dynamo configuration.
+	Your table must look like this:
+
+	field:  type:
+	key     String
+	v       String
+	t       Number
+	i       Boolean
+*/
+export default class LayerDynamo extends Layer {
+
+	static configure(config) {
+		if (!config.tableName) {
+			throw new Error("Must provide tableName");
+		}
+		if (!config.awsConfig) {
+			throw new Error("Must provide awsConfig");
+		}
+		awsConfig = config.awsConfig;
+		tableName = config.tableName;
```

Comment:
The input to `configure` is automatically made available to the instance via `this.opt`.

Reply:
👍